src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/Manager/MediaManagerTest.php

```diff
@@ -520,11 +514,11 @@ public function testMediaPropertiesProvider(): void
         $this->assertSame(['key' => 'value'], $media->getProperties());
     }
 
-    public function provideGetByIds()
+    public static function provideGetByIds(): array
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes